ReactiveCocoa/Swift/Signal.swift

```diff
@@ -154,6 +154,15 @@ public final class Signal<Value, Error: ErrorType> {
 	}
 }
 
+extension Signal {
+	/// Number of observers observing the signal or `nil` if signal is terminated.
+	///
+	/// Property used for Unit Tests.
+	internal var observersCount: Int? {
```

Comment:
Noooooo. We certainly don't want this, as it would mean that we're testing implementation details 👎